gtfs/gtfs_utils/gtfs_utils/core_computations.py

```diff
@@ -329,6 +385,7 @@ def compute_route_stats(trip_stats_subset: pd.DataFrame,
         'end_stop_lat', 'end_stop_lon', 'num_stops', 'start_zone', 'end_zone', 'num_zones', 'num_zones_missing',
         'all_stop_latlon', 'all_stop_code', 'all_stop_id', 'all_stop_desc_city', 'all_start_time', 'all_trip_id',
         'all_stop_name', 'line_type', 'line_type_desc', 'cluster_id', 'cluster_name', 'cluster_sub_desc',
+        'all_trip_id_to_date'
```

Comment:
The field is missing in the function documentation.
It is also missing in `generate_route_stats_aggregation`, so I don't see how this actually works.

Reply:
```
d['all_trip_id_to_date'] = ';'.join(map(str, group['trip_id_to_date'].tolist()))
(`aggregation.py` line 121)
```